Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -378,6 +378,34 @@
       maxCharge: 10000
       startingCharge: 10000
 
+- type: entity
+  name: portable particle decelerator x2
+  suffix: DEBUG
+  parent: WeaponParticleDecelerator
+  id: WeaponParticleDeceleratorX2
+  description: A particle decelerator that simulates 2 people firing them at once.
+  components:
+    - type: Gun
+      fireRate: 1
+      fireCost: 500
```

Comment:
remove fireCost to stop the yaml linter complaining. This is a part of ProjectileBatteryAmmoProvider and not changed anyways.
```suggestion
```